### ISSUE TITLE
Usually f38 failures are for "no alt text" rather than decorative images

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#73]
on sites that have lots of missing alt, this pops up which confuses the advice... usually the advice in this situation is to say add descriptive alt text, rather than mark as decorative... i understand that the check figures that if there is no alt on the first occurrence it is decorative, but that usually isn't the case for images that popup in this failure.

They are usually plain old missing alt text... I've commented this test out until a better way can be found to avoid giving the wrong message for images that require descriptive text, where the first instance of it was missing that required descriptive alt text.

**Comment by irwink** [comment]
Are you typically finding multiple instances of the images on the same page, with the first missing alt text (resulting in me deeming it decorative) ? If this is the case it would be possible to check all instances of the image on the page before deciding if it is decorative or not.  This wont help the case where the first page that has the image uses it as decorative, and subsequent pages use it as non-decorative.

**Comment by DavidMacDonald** [comment]
No, usually I discover this in the summary when I'm dumping a summary into a report and reviewing the categories. I usually have a couple of thousand issues so I don't review any more than a couple of examples of each... but in this case they all seem to be non decorative images that are missing alt text. Not usually a bunch on the same page I don't think... in he last 10 scans I've had it commented out. abut would like to use it if it just identified those as missing alt text. Under WCAãg all images have to have alt text so it is a failure of WCAG if alt is missing whether it should be be alt="" or not.

**Comment by mgifford** [comment]
It would be useful to know what the guideline is within the GC for what is a decorative image.  Is it different than the rough definition here https://webaim.org/techniques/alttext/#decorative
